.gitignore

```diff
@@ -1 +1,2 @@
 dist/
+*.swp
```

Comment:
This is not project specific and should go into your global `.gitignore`.

https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer